### ISSUE TITLE
E1120 for decorated functions

### ISSUE THREAD

**Issue by Virviil** [PyCQA/pylint#1306]
### Steps to reproduce
This is the code
```python
from servers.tasks import taskify

@taskify('make_ml_calculations')
def main(task_payload):
    data = task_payload["test"]
    print(data)
    data = proceed_data(data)
    write_data(data)

def read_data(id):
    return {}

def proceed_data(data):
    return data

def write_data(data):
    pass

if __name__ == "__main__":
    main()
```

Taskify decorator injects payload in main function.
### Expected behavior
Nothing to show

### pylint --version output
21,4,error,E1120:No value for argument 'task_payload' in function call

**Comment by PCManticore** [comment]
What is `taskify` and what is `servers`? Can you provide some code, so that we can see what sort of injection `taskify` does?

**Comment by jcpetruzza** [comment]
Is this not the same as https://github.com/PyCQA/pylint/issues/259 ?

**Comment by Virviil** [comment]
Yes, seams to be the same. Please merge. And please fix.

**Comment by PCManticore** [comment]
Closed as duplicate.
